pkgs/desktops/gnome-3/extensions/taskwhisperer/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, taskwarrior }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-taskwhisperer-${version}";
+  version = "11";
+
+  src = fetchFromGitHub {
+    owner = "cinatic";
+    repo = "taskwhisperer";
+    rev = "v${version}";
+    sha256 = "1g1301rwnfg5jci78bjpmgxrn78ra80m1zp2inhfsm8jssr1i426";
+  };
+
+  propagatedBuildInputs = [ taskwarrior ];
+
+  uuid = "taskwhisperer-extension@infinicode.de";
+
+  installPhase = ''
+    sed -i "s,\['task',\['${taskwarrior}/bin/task',g" taskService.js
+
+    mkdir -p $out/share/gnome-shell/extensions/${uuid}
+    cp *.js $out/share/gnome-shell/extensions/${uuid}
+    cp -r extra $out/share/gnome-shell/extensions/${uuid}
+    cp -r icons $out/share/gnome-shell/extensions/${uuid}
+    cp -r locale $out/share/gnome-shell/extensions/${uuid}
+    cp -r schemas $out/share/gnome-shell/extensions/${uuid}
+    cp -r translation $out/share/gnome-shell/extensions/${uuid}
```

Comment:
This probably is not needed for installation (https://extensions.gnome.org/extension-data/taskwhisperer-extension%40infinicode.de.v10.shell-extension.zip). Ugh, they should just provide a makefile.